content/posts/irods-development-update-august-2022.md

```diff
@@ -0,0 +1,313 @@
+
+To start things off, TRiRODS was a success. Our interns did a wonderful job on their projects and presentations. If you're interested in hearing about their experience, you can view their talks online at https://irods.org/trirods/.
+
+The iRODS 4.3 server has received some improvements. The main iRODS server process now supports live reloading of **server_config.json**. Another notable improvement is that the `remote()` microservice now honors the `INST_NAME` hint. This will allow users to not only target a specific server, but also target a specific Rule Engine Plugin, just like the `delay()` microservice. Work has also been done to disallow downgrades of the iRODS user account used to manage an iRODS server. This change helps to prevent users and applications from breaking a deployment.
+
+We are also working with SURF to make PAM authentication the default scheme for authenticating with an iRODS server. Leveraging the [PAM interactive](https://github.com/stefan-wolfsheimer/irods/tree/interactive_pam) work SURF is doing means iRODS can support different authentication schemes all while providing a more dynamic conversation with clients. Please see the [Authentication Working Group minutes](https://github.com/irods-contrib/irods_working_group_authentication) for additional details.
+
+On the client-side, Metalnx is seeing some improvements. Metalnx now honors the `compute.checksum` property in metalnx.properties. Data objects can now be downloaded using tickets. We are working on a few more bug fixes, so you can expect a new release soon.
+
+The C++ REST API has received some big enhancements as well. Two new endpoints have been added to the API, `/metadata` and `/logicalpath`. As hinted by the name, `/metadata` is all about manipulating metadata. It is backed by the atomic metadata operations API. The `/logicalpath` endpoint gives users a way to interact with data objects and collections outside of just reading and writing to them. With this endpoint, users will be able to rename and unlink objects.
+
+Several plugins have been improved this past month. The S3 Resource Plugins for the 4.3 series and 4.2 series now support Amazon Glacier / Deep Archive. Not only that, but several bugs have been fixed as well (e.g. support for `itouch` and `istream --append`).
+
+Work on the Indexing Capability Plugin has been merged. It now handles configuration properties that accept integers and strings more gracefully. It also handles `iput --metadata` without any issues. Work to support an SSL-capable elasticlient is in full swing too.
+
+Work to replace all use of the Jansson JSON library in the Audit AMQP Rule Engine Plugin has been going well. Aside from the plugin, the [ELK stack](https://github.com/irods/contrib/tree/main/irods_audit_elk_stack) has received some major updates. All components in the project have been updated to a supported version. The project now works with the iRODS 4.3 series and 4.2 series.
```

Comment:
‘used for training’

Reply:
Is that ...
```
The project used for training now works with the iRODS 4.3 series and 4.2 series.
```
or
```
The project now works with the iRODS 4.3 series and 4.2 series used for training.
```

Reply:
…`, used for training, has received some major updates.`

Reply:
Will change.

Reply:
Done